server/middleware/authorize/authorizeRequest.js

```diff
@@ -16,7 +16,7 @@ const defaultConvertRequestToAction = (method) => {
 }
 
 const authorizeActions = (subject, convertRequestToAction = defaultConvertRequestToAction) => (req, res, next) => {
-  const action = convertRequestToAction(req.method)
```

Comment:
and req.method passed in here.  so where is the issue?

Reply:
The issue is in https://github.com/voluntarily/vly2/blob/master/server/api/opportunity/opportunity.routes.js
 with the convertRequestToAction function which has been coded as taking in an req object but is being passed a string.

Reply:
I agree its unclear because the function is called convert request to action so it would be expected to accept a request.   Otherwise, it could be called convertMethodToAction. 

Usually, I would say the function should be a simple as possible and would therefore just use the method string.  However, it's possible that certain actions might need more information from the request to make the decision. In this case, there is value in passing in the req and letting the implementor decide what to use. 

So your changes are correct. the functions take a req.  please update the unit tests to pass in an object { method: 'GET' } as a mock request. 

e.g in AuthoriseRequest.spec.js 66, 70 and 74.

Reply:
Yep sounds good. Thanks for the review.